web-client/src/presenter/actions/setProgressForFileUploadAction.ts

```diff
@@ -0,0 +1,118 @@
+import { FileUploadProgressMapType } from '../../../../shared/src/business/entities/EntityConstants';
+import {
+  dateStringsCompared,
+  formatNow,
+} from '@shared/business/utilities/DateHandler';
+import { state } from '@web-client/presenter/app.cerebral';
+
+export const setProgressForFileUploadAction = ({
+  get,
+  props,
+  store,
+}: ActionProps<{
+  files: any;
+}>): {
+  fileUploadProgressMap: FileUploadProgressMapType;
+} => {
+  const { files } = props;
+  const loadedAmounts: Record<string, number> = {};
+  const startTime = formatNow();
+  const sizeOfFiles: Record<string, number> = {};
+
+  Object.keys(files).forEach(key => {
+    if (!files[key]) return;
+    if (Array.isArray(files[key])) {
+      sizeOfFiles[key] = files[key][0].size;
+      //     files[key].forEach((file, index) => {
+      //       totalSizes[`${key}-${index}`] = file.size;
+      //     });
```

Comment:
for discussion again: should these comments be removed?

Reply:
Yeah, i think we can remove the array logic. It was removed everywhere else